depends/packages/expat.mk

```diff
@@ -1,12 +1,13 @@
 package=expat
-$(package)_version=2.2.7
-$(package)_download_path=https://github.com/libexpat/libexpat/releases/download/R_2_2_7/
-$(package)_file_name=$(package)-$($(package)_version).tar.bz2
-$(package)_sha256_hash=cbc9102f4a31a8dafd42d642e9a3aa31e79a0aedaa1f6efd2795ebc83174ec18
+$(package)_version=2.4.1
+$(package)_download_path=https://github.com/libexpat/libexpat/releases/download/R_2_4_1/
+$(package)_file_name=$(package)-$($(package)_version).tar.xz
+$(package)_sha256_hash=cf032d0dba9b928636548e32b327a2d66b1aab63c4f4a13dd132c2d1d2f2fb6a
 
 define $(package)_set_vars
   $(package)_config_opts=--disable-shared --without-docbook --without-tests --without-examples
-  $(package)_config_opts += --disable-dependency-tracking --enable-option-checking
+  $(package)_config_opts += --disable-dependency-tracking --without-xmlwf
```

Comment:
A note for reviewers: the `--without-xmlwf` option was [added](https://github.com/libexpat/libexpat/blob/08d29debaee59fc44f470e2392b78820015afcf2/expat/Changes#L479) in Release 2.2.4.